cmd/server/cadence/server.go

```diff
@@ -119,6 +119,7 @@ func (s *server) startService() common.Daemon {
 	dc := dynamicconfig.NewCollection(params.DynamicConfig, params.Logger)
 
 	svcCfg := s.cfg.Services[s.name]
+	params.Service = svcCfg
```

Comment:
I find this confusing. I believe what is happening here is in this file we set svcConfig. Then in each service (frontend, history, matching, worker). We update a field of this service config (maxQPS). Then we access this reference when we initialize the bean when we initialize the persistence factory? 

Another option is don't add anything to bootstrap params and just pass the dynamic config value to the bean creation when each service bootsup.